CMakeLists.txt

```diff
@@ -55,7 +55,20 @@ ADD_DEFINITIONS(-DPROJECT_SOURCE_DIR=\\"${CMAKE_SOURCE_DIR}\\"
                 -DPROJECT_BINARY_DIR=\\"${CMAKE_BINARY_DIR}\\")
 
 # add the 'lib' dir to cmake's module search path
-SET(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/lib/")
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/lib/")
+
+# Cogutil
+FIND_PACKAGE(CogUtil)
```

Comment:
According to [find-package documentation](https://cmake.org/cmake/help/latest/command/find_package.html#command:find_package) should be:

    FIND_PACKAGE(CogUtil CONFIG)

to skip searching FindCogUtil.cmake and look for CogUtilConfig.cmake

Reply:
updated